nupic/research/frameworks/pytorch/imagenet/experiment_utils.py

```diff
@@ -345,3 +349,161 @@ def create_model(model_class, model_args, init_batch_norm, device,
             model = hook(model, **kwargs) or model
 
     return model
+
+
+def get_free_port():
```

Comment:
Since there are several functions which belong together, could you put these functions in a file more specific to this  use case? For example `experiment_search.py`?

Reply:
Moved to new `experiment_search` file